worlds/stardew_valley/options/options.py

```diff
@@ -674,6 +693,33 @@ class TrapItems(Choice):
     option_nightmare = 5
 
 
+trap_default_weight = 100
+
+
+class TrapDistribution(OptionCounter):
+    """
+    Specify the weighted chance of rolling individual traps when rolling random filler items.
+    The average filler item should be considered to be "100", as in 100%.
+    So a trap on "200" will be twice as likely to roll as any filler item. A trap on "10" will be 10% as likely.
+    You can use weight "0" to disable this trap entirely. The maximum weight is 1000, for x10 chance
+    """
+    internal_name = "trap_distribution"
+    display_name = "Trap Distribution"
+    default_weight = trap_default_weight
+    min = 0
+    max = 1000
+    schema = Schema({
+        trap_data.name: And(int, lambda n: 0 <= n <= 1000)
+        for trap_data in items_by_group[Group.TRAP]
+        if Group.DEPRECATED not in trap_data.groups
+    })
+    default = {
+        trap_data.name: trap_default_weight
+        for trap_data in items_by_group[Group.TRAP]
+        if Group.DEPRECATED not in trap_data.groups
+    }
```

Comment:
A couple things:
1. You don't need the schema
2. You _probably_ want to define valid_keys, so it can display on webhost since it currently does not
3. All of these weights are 100?
```suggestion
    valid_keys = frozenset({
        trap_data.name
        for trap_data in items_by_group[Group.TRAP]
        if Group.DEPRECATED not in trap_data.groups
    })
    default = {
        trap_data.name: trap_default_weight
        for trap_data in items_by_group[Group.TRAP]
        if Group.DEPRECATED not in trap_data.groups
    }
```

Reply:
Yes, all weights start at 100. What makes you think I wouldn't want that?

Reply:
It just seems odd to me that you're grabbing the default value when they're all 100. But I guess it's just future proofing

Reply:
I'm just used to preferring to make a constant rather than a magic number I guess

Reply:
Oh I misread that 🤦 yeah you're good